tests/ctst/features/azureArchive.feature

```diff
@@ -121,10 +121,13 @@ Feature: Azure Archive
     And object "obj-2" should have the same data
     And object "obj-2" should have the tag "tag2" with value "value2"
     And object "obj-2" should have the user metadata with key "x-amz-meta-123" and value "456"
+    When i wait for <restoreDays> days
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-azure-archive"
```

Comment:
Looking at the sentence, "transitioned" does not really make sense here: we are no transitioning, and there is no transition at this stage.

We should find another name, maybe "cold" would be best (thinking of the automaton: the object can indeed either be "hot", "transitioning" (to cold), "cold", "restoring", or "restored")